### PR TITLE
[WIP][ADD] base_geoengine_jsonify: support of geo_point json exports

### PR DESCRIPTION
**Pull request by len-foss**
Dependent on https://github.com/OCA/server-tools/pull/1894 refactoring to actually work.

### CODE REVIEW

File: base_geoengine_jsonify/models/models.py
Comment (lmignon) [inline base_geoengine_jsonify/models/models.py]: @len-foss The same logic could be applied to all the geo fields non? 

```python

from  odoo.addons.base_geoengine.fields import GeoField

....

if res and isinstance(field, GeoField):
    res = tools.safe_eval(geojson.dumps(res))
...

```

**Review by lmignon — APPROVED** [review]
Great!
LGTM once OCA/server-tools#1894  is merged

File: base_geoengine_jsonify/models/models.py
Comment (sbidoul) [inline base_geoengine_jsonify/models/models.py]: If `geojson.dumps` produces a json str, is it not better to parse it with `json.loads` ?

**Comment by yvaucher** [comment]
OCA/server-tools#1894 was merged on Jan 21

**Review by yvaucher — APPROVED** [review]
LGTM

Looks interesting

File: base_geoengine_jsonify/models/models.py
Comment (yvaucher) [reply base_geoengine_jsonify/models/models.py]: This seems to have been resolved with:
 
 ``` python
           # we eval it as dumps gives a str, we need a point for json.dumps
            res = geojson.loads(geojson.dumps(res))
```

Seems weird, waiting on travis results (don't know what to expect from the content of `res` in this context)

**Comment by yvaucher** [comment]
Just relaunched the CI

File: base_geoengine_jsonify/models/models.py
Comment (yvaucher) [inline base_geoengine_jsonify/models/models.py:14]: Let's be a bit more generic, geojson.dumps should support multiline string and polygones too.

```suggestion
        """Adds support for geometry fields."""
```

File: base_geoengine_jsonify/readme/DESCRIPTION.rst
Comment (yvaucher) [inline base_geoengine_jsonify/readme/DESCRIPTION.rst:1]:
```suggestion
Automatically export geometry fields as geojson in json exports.
```

File: base_geoengine_jsonify/models/models.py
Comment (yvaucher) [inline base_geoengine_jsonify/models/models.py:17]:
```suggestion
            # we eval it as dumps gives a str, we need a geometry for json.dumps
```

File: base_geoengine_jsonify/readme/USAGE.rst
Comment (yvaucher) [inline base_geoengine_jsonify/readme/USAGE.rst:2]:
```suggestion
If you add a geometry to a json export, the point will be converted automatically
```

File: base_geoengine_jsonify/__manifest__.py
Comment (yvaucher) [inline base_geoengine_jsonify/__manifest__.py:4]:
```suggestion
    "name": "Export geometry as GeoJSON",
```

**Review by yvaucher — APPROVED** [review]
Nice, few suggestion to make the module a bit more generic in the terminology as GeoPoint is not the only geometry that can be handled. I assume it would work too, to create a GeoJSON for a Polygon or a LineString.

**Pull request closed without merging**
